### PR TITLE
add way to jump to page section on focus area pages

### PR DESCRIPTION
**Pull request by davidlange6**
The focus area pages are getting long. This pr adds a simple menu at the top  - for IA it looks like this:

image

This did mean breaking up the video list making include into two pieces - one to select and one to display the videos

### CODE REVIEW

**Review by henryiii — APPROVED** [review]
LGTM

File: _layouts/focus-area.html [hunk 1]
```diff
@@ -6,17 +6,39 @@
...
+{% if pres_list.size > 0 %}
+{% assign quicklinks = quicklinks | append: '- <a href="#presentations"> Presentations</a> ' %}
+{% endif %}
+{% if pubs.size > 0 %}
+{% assign quicklinks = quicklinks | append: '- <a href="#publications"> Publications</a> ' %}
+{% endif %}
+{% if selected_videos.size > 0 %}
+{% assign quicklinks = quicklinks | append: '- <a href="#videos"> Videos</a> ' %}
+{% endif %}
+{{ quicklinks }}
```
Comment (henryiii) [inline _layouts/focus-area.html]:
```suggestion
{% assign quicklinks "" = split: "," %}
{% if fa_fellows.size > 0 %}
{% assign quicklinks = quicklinks | push: '<a href="#fellows"> Fellows</a>' %}
{% endif %}
{% if sorted.size > 0 %}
{% assign quicklinks = quicklinks | push: '<a href="#projects"> Projects</a>' %}
{% endif %}
{% if pres_list.size > 0 %}
{% assign quicklinks = quicklinks | push: '<a href="#presentations"> Presentations</a>' %}
{% endif %}
{% if pubs.size > 0 %}
{% assign quicklinks = quicklinks | push: '<a href="#publications"> Publications</a>' %}
{% endif %}
{% if selected_videos.size > 0 %}
{% assign quicklinks = quicklinks | push: '<a href="#videos"> Videos</a>' %}
{% endif %}
Jump to: {{ quicklinks | join: " - " }}
```

**Comment by davidlange6** [comment]
ha - whoops -Liquid Exception: Liquid syntax error (line 17): Syntax Error in 'assign' - Valid syntax: assign [var] = [source] in /home/runner/work/iris-hep.github.io-source/iris-hep.github.io-source/_layouts/focus-area.html

@henryiii  if you see the problem your change introduced, go ahead and fix - otherwise I will tomorrow

File: _layouts/focus-area.html [hunk 2]
```diff
@@ -6,17 +6,39 @@
...
 
+{% assign page_focus_area = page.name | basename %}
+{% assign sorted = site.pages | where: "pagetype", "project" | where_exp: "item", "item.focus-area contains page_focus_area" | smart_title_sort %}
+{% assign fa_team = site.data.people | where_exp: "item", "item.focus-area contains page_focus_area and item.active and item.hidden != true" | last_name_sort: "name" %}
+{% assign fa_fellows = site.pages | sort_natural: 'title' | where_exp: "item", "item.pagetype == 'fellow' and item.focus-area contains page_focus_area" %}
+{% assign pres_list = site.data['sorted_presentations'] | where_exp: "item", "item.focus-area contains page_focus_area"  %}
+{% assign pubs = site.data['sorted_publications'] | where_exp: "item", "item.focus-area contains page_focus_area" %}
+{% include get_selected_videos.html nvideos=6 focus_areas=page_focus_area %}
+
+{% assign quicklinks "" = split: "," %}
```
Comment (henryiii) [inline _layouts/focus-area.html]:
```suggestion
{% assign quicklinks = "" | split: "," %}
```

File: _layouts/focus-area.html [hunk 3]
```diff
@@ -6,17 +6,39 @@
...
+{% if pres_list.size > 0 %}
+{% assign quicklinks = quicklinks | push: '<a href="#presentations"> Presentations</a>' %}
+{% endif %}
+{% if pubs.size > 0 %}
+{% assign quicklinks = quicklinks | push: '<a href="#publications"> Publications</a>' %}
+{% endif %}
+{% if selected_videos.size > 0 %}
+{% assign quicklinks = quicklinks | push: '<a href="#videos"> Videos</a>' %}
+{% endif %}
+Jump to: {{ quicklinks | join: " - " }}
```
Comment (henryiii) [inline _layouts/focus-area.html]:
```suggestion
{% if quicklinks.size > 0 %}
  Jump to: {{ quicklinks | join: " - " }}
{% endif %}
```

**Pull request merged**